workflow/packer/packer.go

```diff
@@ -86,13 +108,18 @@ func compressWorkflow(wf *wfv1.Workflow) error {
 		return err
 	}
 	if large {
-		compressedSize, err := getSize(wf)
+		compressedSize, err1 := getSize(wf)
+		nodesSize, err2 := getNodeStatusSize(wf)
 		wf.Status.CompressedNodes = ""
 		wf.Status.Nodes = nodes
-		if err != nil {
+		if err1 != nil {
```

Comment:
Move this the after line 111 and then remove back to err?

Reply:
Code updated, I have confirmed that if an error occurs or is too large, the status will be restored and an error will be returned.